lib/grocery_list.rb

```diff
@@ -0,0 +1,42 @@
+class GroceryList
+  attr_accessor :owner, :items
+
+  def initialize(args)
+    @items = args[:items]
+    puts "Do you see me here in grocery_list?"
+    @owner        = args[:owner]
+  end
+
+  def owner
+    @owner
+  end
+
+  def owner=(new_owner)
+    @owner = new_owner
+  end
```

Comment:
Both this getter and setter are duplicates because you have `attr_accessor :owner`